src/Form/EntryForm.php

```diff
@@ -51,13 +59,14 @@ public function getFormID() {
    */
   public function buildForm(array $form, FormStateInterface $form_state, AccountInterface $user = null) {
     // Check flood tables.
-    //@TODO Reimplement Flood Controls.
+
```

Comment:
Please note - I'm just asking why, not suggesting they should be. If you implemented flood controls, maybe we should consider removing the commented out code